build.sh

```diff
@@ -68,6 +68,14 @@ export NUGET_HTTP_CACHE_PATH="$REPOROOT/packages"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_cli
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
+# Some things depend on HOME and it may not be set. We should fix those things, but until then, we just patch a value in
+if [ -z "$HOME" ]; then
+    export HOME="$REPOROOT/.home"
+
+    [ ! -d "$HOME" ] || rm -Rf $HOME
```

Comment:
I have an extremely strong feeling that CliCommandLineParser just copied it from

https://github.com/dotnet/cli/blob/master/build.sh#L19-L25

which came in over a year ago here:
https://github.com/dotnet/cli/commit/34b0b688f7db70003514e0aefbee47868e6d5f71
with the commit message `patch up some final missing CI build things`